tests/integration/framework/process/workflow/workflow.go

```diff
@@ -97,6 +98,7 @@ spec:
 				dopts = append(dopts, daprdOpt.opts...)
 			}
 		}
+		dopts = append(dopts, daprd.WithAppID(fmt.Sprintf("app%d", i)))
```

Comment:
Was this just for testing purposes? The framework should automagically create daprd's with a unique appid each time anyways. 
```suggestion
```

Reply:
I find it easier to debug when apps have a readable name. Using UUIDs is fine when everything is green, but when we have to look at the logs it's hard to work with UUIDs when apps call other apps. When I was implementing this feature I had to edit the logs, replacing those UUIDs with `appN`, so I thought it would be good to change this line so logs are easier to process.

Reply:
It is intentional to use UUIDs for app IDs because they ultimately get made as mDNS entries where if they are reused, can cause DNS misses due to caching if the same name is used.